app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container my-5">
+  <div class="row justify-content-center">
+    <div class="col-md-6 col-lg-5">
+      <div class="card p-4 shadow-sm">
+        <h2 class="mb-3 text-second font-semibold text-center">Sign up</h2>
+        <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: {class: 'row g-3'}) do |f| %>
+          <%= render "devise/shared/error_messages", resource: resource %>
+          <div class="col-12 mb-4">
+            <%= f.text_field :name, id: "name", class: "form-control", placeholder: "Full Name" %>
+          </div>
+          <div class="col-12 mb-4">
+            <%= f.email_field :email, id: "email", class: "form-control", placeholder: "Email" %>
+          </div>
+          <div class="col-12 mb-4">
+            <%= f.password_field :password, id: "password", class: "form-control", placeholder: "Password" %>
+          </div>
+          <div class="col-12 mb-4">
+            <%= f.password_field :password_confirmation, id: "password_confirmation", class: "form-control", placeholder: "Confirm password" %>
+          </div>
+          <div class="col-12 mb-4">
+            <%= f.button "Sign up", type: "submit", class: "btn btn-primary w-100", 'data-ripple-light': true %>
+          </div>
+          <div class="col-12 text-center">
+            <%= render "devise/shared/links" %>
+          </div>
```

Comment:
- [ OPTIONAL ] Great job so far but I think you missed to style your sign up page the same as the given design so I highly recommend to style it the same since it is very important to have the same design as the givem design but I am assign it as optional because the previous code reviewer doesn't mention it 😊 but kindly fix the styles here ✨✅♻🙏🏻🙏🏻

## There are alot of different things the nav contain the button for sign up  and arrow  in the original design  ✅
- Your design 
![image](https://user-images.githubusercontent.com/58553711/221363137-517d7942-9844-4431-afea-cc3ba9302b3e.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/221363221-751b4c25-9567-4948-93f4-52f01aa2c15e.png)

Reply:
I'm limited on time but I will work on the page.